### PR TITLE
adding practice programs

### PR DESCRIPTION
**Pull request by Shruti1-GitHub**
hiyaaaa

### CODE REVIEW

File: src/Practice/Fibonacii.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class Fibonacii {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		int n1=0; int n2=1;
+		int n3; int cnt=10;
+		System.out.print(n1+" "+n2);
+		
+		for(int i=2;i<cnt;i++)
+		{
```
Comment (srgrj) [inline src/Practice/Fibonacii.java:12]: Good Logic

File: src/Practice/Even_Number.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		
+		int a=11;
+		if(a%2==0)
+		{
+			System.out.println(a+" is even number");
+		}else
+		{
```
Comment (srgrj) [inline src/Practice/Even_Number.java:15]: Is this a basic test program?

**Review by srgrj — COMMENTED** [review]
Added comments

File: src/Practice/Even_Number.java (hunk 2 above)
Comment (Shruti1-GitHub) [reply src/Practice/Even_Number.java:15]: yes

**Review by srgrj — APPROVED** [review]
Looks Good To ME

**Pull request merged**